test_accounts/legacy_appeals_test_accounts.md

```diff
@@ -0,0 +1,27 @@
+# Sandbox Test Accounts for Legacy Appeals API (v0)
+
+This document provides test account information for the [Legacy Appeals API (v0)](https://developer.va.gov/explore/api/legacy-appeals/docs?version=current). The API returns synthetic VA records for these test accounts in the sandbox environment.
+
+## Authentication and Authorization
+
+The authentication model for the [Legacy Appeals API (v0)](https://developer.va.gov/explore/api/legacy-appeals/docs?version=current) uses OAuth 2.0/OpenID Connect. The following authorization models are supported:
+- [Authorization code flow](https://developer.va.gov/explore/api/legacy-appeals/authorization-code)
+- [Client Credentials Grant (CCG)](https://developer.va.gov/explore/api/legacy-appeals/client-credentials)
+
+Login information for all test accounts can be found on the [developer portal](link-will-go-here-once-page-created).
+
+## Veteran Test Accounts
+
+These accounts can be used to test various data and response scenarios for the `GET /legacy-appeals` endpoint. 
+
+An `icn` parameter is required when making a request: it should match the value in the ICN column.
+
+| Sandbox Response          | ICN               | ID.me Username                 | Login.gov Username        | First Name | Last Name |
+| :------------------------ | :---------------- | :----------------------------- | :------------------------ | :--------- | :-------- |
+| 2 Legacy Appeals          | 1012832025V743496 | va.api.user+idme.025@gmail.com | va.api.user+025@gmail.com | Wesley     | Ford      |
```

Comment:
Melinda suggested listing the responses like this:
Legacy Appeals (2)
Legacy Appeal (1)